src/icons/fileIcons.ts

```diff
@@ -422,6 +422,7 @@ export const fileIcons: FileIcons = {
 
         'docker-compose.yml',
         'docker-compose.override.yml',
+        'docker-compose.override.yml.template',
```

Comment:
Thanks, it's a good addition. I have thought a little further and had the idea of creating a generic icon for all `*.template` files like this:

<img src="https://user-images.githubusercontent.com/12248527/128774113-ca0a1b0b-a1c4-4d96-94ca-6c045bebe047.png" alt="template icon preview" width="400px"/>

Any opinions from your side? 
@SNDST00M what do you think about this? 

I've used the [hidden icon](https://materialdesignicons.com/icon/file-hidden) from the Material Design Icons here and `#90A4AE` as color.

Reply:
Usage seems distinct to `dist` so it seems all right to me.